data/wanderer/wanderers start.txt

```diff
@@ -1990,9 +2000,10 @@ mission "Wanderers Hai Assistance 2"
 				goto threaten
 			label attack
 			`	"I will not authorize preemptive violence against our own flesh and blood," says one of the elders. "We only defend ourselves against them, never willingly initiate an attack."`
+			`	Teeneep and another Elder who seems to be in her corner both chitter the Hai equivalent of "Hear Hear" to this, though she is being very careful with her contributions.`
 			label threaten
-			`	"Perhaps it would be enough merely to threaten them," says another of the elders. "We could gather a massive war fleet above Cloudfire. The Unfettered would need to muster a matching fleet above Darkcloak. It would draw many of their warships away from the alien territory."`
-			`	After well over an hour of further discussion, they reach consensus that that is their best option. As you leave the meeting, Sayari says, "And now, I would like to visit the Unfettered themselves. Perhaps we can talk sense into them."`
+			`	"Perhaps Teeneep is right and it would be enough merely to threaten them," says another of the elders. "We could gather a massive war fleet above Cloudfire. The Unfettered would need to muster a matching fleet above Darkcloak. It would draw many of their warships away from the alien territory."`
```

Comment:
I've only noticed now that there's probably a lot of places where 'Elder' isn't capitalised. Is a PR necessary for this?

Reply:
Would like a smidgen of clarity from @W1zrad on the capitalisation of Elders - then I'll just standardise it in this PR.

Reply:
As far as I'm concerned it's a title like Captain or Senator or Member of Parliament. HR usually capitalises 'Elder' while earlier mentions (I guess by MZ) don't.

Reply:
I will check through for standardising to that in a bit then. :)